org.agileware.natural.cucumber/src/org/agileware/natural/cucumber/generator/CucumberGeneratorMWE.mwe2

```diff
@@ -0,0 +1,31 @@
+module org.agileware.natural.cucumber.generator.CucumberGeneratorMWE
```

Comment:
This was auto-generated by eclipse. Should it be removed?

Reply:
no need to remove it